client/store/thunks.js

```diff
@@ -9,3 +9,17 @@ export const login = (email, password) => {
       .then(user => dispatch(actions.getUser(user)))
   }
 }
+
+export const checkForUserThunk = () => {
```

Comment:
can we name this something more specific? checkIfUserLoggedIn maybe - just to make it more readable. We're going to end up with like 1000 thunks I think.